requirements/base.in

```diff
@@ -1,6 +1,7 @@
 # Core requirements for using this package
 -c constraints.txt
 
+edx-opaque-keys
```

Comment:
I added opaque-keys as a dependency for the new unit tests. Normally I wouldn't add a base dependency just for a couple tests, but we anticipate to make the repo depend on opaque-keys soon anyway, so this seemed like a good step:
 * https://github.com/openedx/XBlock/issues/707
 * https://github.com/openedx/XBlock/issues/708